### PR TITLE
Hit the API

### PR DESCRIPTION
**Pull request by Diegogagan2587**
# Implemented Features:
- Create a new Game
- Save score using API when clicking on submit.
- Retrieve Score when clicking on Refresh using API

### CODE REVIEW

File: src/modules/create_new_game.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+async function getGameID(object) {
```
Comment (whiz25) [inline src/modules/create_new_game.js]: - Please use `arrow` syntax to define functions not the `function` keyword.

File: src/modules/create_new_game.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  const newGame = await fetch(urlAPI, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify({
+      name: 'Diego Vidal Game',
+    }),
+  })
+    .then((response) => response.json()); // then get the ID of the new game
```
Comment (whiz25) [inline src/modules/create_new_game.js]: - Please make use of `async` `await`

File: src/modules/get_scores.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { testGameID as gameID } from './create_new_game.js';
+import display from './display.js';
+
+const urlAPI = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameID}/scores`;
+
+async function getScores() {
```
Comment (whiz25) [inline src/modules/get_scores.js]: - Again use `arrow` syntax here anywhere where there's a function

**Review by whiz25 — CHANGES_REQUESTED** [review]
Hi @Diegogagan2587, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Professional readme template used 
- Logic for fetching scores with API implemented

## Required Changes ♻️
_Check the comments under the review._
- You have faling linter checks regarding `webhint`. Much as it's an external issue, I found two links that may help you fix the error. Check below
[Link1](https://webhint.io/docs/user-guide/troubleshoot/summary/)
[Link2](https://github.com/webhintio/hint/issues/4830)
- Your app failed to run this is the error I got



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/create_new_game.js (hunk 1 above)
Comment (Diegogagan2587) [reply src/modules/create_new_game.js]: Thanks, Done!

File: src/modules/get_scores.js (hunk 3 above)
Comment (Diegogagan2587) [reply src/modules/get_scores.js]: Done!

File: src/modules/create_new_game.js (hunk 2 above)
Comment (Diegogagan2587) [reply src/modules/create_new_game.js]: Done!

File: src/modules/submit_score.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { testGameID as gameID } from './create_new_game.js';
+
+function submitScore(event) {
```
Comment (Zuhaib042) [inline src/modules/submit_score.js:3]: - [OPTIONAL] Kindly consider using the ES6 arrow syntax for functions to follow the best practices.

**Review by Zuhaib042 — APPROVED** [review]
Hi @Diegogagan2587 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Descriptive PR
✅ Professional Readme
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/submit_score.js (hunk 4 above)
Comment (Diegogagan2587) [reply src/modules/submit_score.js:3]: I miss that one, Thanks for the suggestion, Will do it!

**Pull request merged**